app/core/tests/views/test_submission_views.py

```diff
@@ -212,6 +213,6 @@ def test_download_input_file(client, user, benzene_from_mol):
     response = client.get(f"/download_input/{input_value.pk}/")
     assert response.status_code == 200
     assert isinstance(response, FileResponse)
-    assert (
-        response.headers["Content-Disposition"] == 'inline; filename="ChEBI_16716.mdl"'
+    assert re.match(
+        'inline; filename="ChEBI_16716.*.mdl"', response.headers["Content-Disposition"]
```

Comment:
I might have missed it but why the check for this pattern instead of just the file name?